lib/aggregate-messages.js

```diff
@@ -1,3 +1,5 @@
+// trigger changelog failure
```

Comment:
This guy needs clean up

Reply:
Cleaned up in this commit: https://github.com/cerner/terra-aggregate-translations/pull/6/commits/aad64411639b7eea971a457cc92a0efb6f8d6745